src/ui.js

```diff
@@ -1,59 +1,77 @@
 /* eslint-disable prettier/prettier */
-import { pedirListaPokemon, pedirPokemon } from "./backend.js";
-import { capitalizarPalabra } from "./helper/helper.js";
+import {
+  fetchPokemon,
+  fetchPokemonList,
+  previousPage,
+  nextPage,
+} from "./servicio.js";
+import { capitalizeWord } from "./helper/helper.js";
 
-const cantidadHabilidadesPokemon = 6;
+const amountOfPokemonAbilities = 6;
+export function initialize() {
+  createPokemonList(0);
+  printPokemon("bulbasaur");
+}
 
-export async function crearPokemon(pokemon) {
+export async function printPokemon(pokemon) {
   pokemon = pokemon.toLowerCase();
-  const pokemonJSON = await pedirPokemon(pokemon);
-  armarPokemon(pokemonJSON);
+  const pokemonJSON = await fetchPokemon(pokemon);
+  buildPokemon(pokemonJSON);
 }
 
-function borrarListaPokemon() {
-  $(".listaPokemon").remove();
+function deletePokemonList() {
+  $(".pokemon-list").remove();
 }
 
-export async function crearListaPokemon(offset) {
-  borrarListaPokemon();
-  const listaPokemonesJSON = await pedirListaPokemon(offset);
+export async function createPokemonList(offset) {
+  deletePokemonList();
+  const pokemonListJSON = await fetchPokemonList(offset);
 
-  cambiarListaPokemon(listaPokemonesJSON);
-  aniadirOnclickAElementos();
+  changePokemonList(pokemonListJSON);
+  setupPokemonEventHandler();
 }
 
-$(".ubicacion-grid-00").append(
-  `<button class="btn btn-success boton-anterior">Anterior</button>`,
-  `<button class="btn btn-success boton-siguiente">Siguiente</button>`,
+$(".location-grid-00").append(
+  `<button class="btn btn-success previos-page-button">Previous</button>`,
+  `<button class="btn btn-success next-page-button">Next</button>`,
   `<br/>`
 );
 
-function armarPokemon(pokemonJSON) {
+function buildPokemon(pokemonJSON) {
   $(".card-img-top").attr("src", `${pokemonJSON.sprites.front_default}`);
-  $(".nombre-pokemon").text(capitalizarPalabra(`${pokemonJSON.species.name}`));
-  for (let i = 0; i < cantidadHabilidadesPokemon; i++) {
-    $(`.habilidad${i}`).text(
+  $(".pokemon-name").text(capitalizeWord(`${pokemonJSON.species.name}`));
+  for (let i = 0; i < amountOfPokemonAbilities; i++) {
+    $(`.ability${i}`).text(
       `${pokemonJSON.stats[i].stat.name} + ${pokemonJSON.stats[i].base_stat}`
     );
   }
 }
 
-function cambiarListaPokemon(pokemones) {
-  pokemones.results.forEach((elemento) => {
-    $(".ubicacion-grid-00").append(
-      `<button class="btn btn-primary d-flex flex-nowrap border listaPokemon">${capitalizarPalabra(
-        elemento.name
+function changePokemonList(pokemon) {
```

Comment:
renderPokemonList, y lo que toma como parámetor no es 1 solo pokemon, no? Entonces ponele otro nombre.

Reply:
acá, si hacés los cambios de arriba, estarías tomando un array (una lista) de objetos que describen el nombre de un pokemon, entonces:

```suggestion
function renderPokemonList(pokemonList) {
```

Fijate que ahora tiene más sentido, estás llamando a la función que renderea un pokemon list, y como parámetro qué le pasás? Un pokemon list, el que se va a renderear.